similarity/src/test/java/edu/illinois/cs/cogcomp/entitySimilarity/compare/LevensteinTester.java

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
I removed this test. This file is copied from other package. For Levenstein , I just call from https://github.com/CogComp/cogcomp-nlp/blob/365172b5b64c70430e248dd692e823c66645cde1/core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/algorithms/LevensteinDistance.java